### DISCUSSION TITLE
Extra Data File for the Converter

### DISCUSSION THREAD

**Discussion by LLRiebel** [rsachetto/MonoAlg3D_C#55] [General]
Hi, I saw the new functionality to provide an extra data file to the alg -> vtk converter. Could you provide an example, as I am unsure which format this extra data file needs. Thanks!

**Comment by rsachetto** [comment]
It is possible to provide a configuration file and map each ALG mesh column to a scalar field in the VTK output.
For instance, if you provide the following configuration file to the tool (by using the -c flag):
``` ini
[scar]
n_components=1
column_index=7

[fibers]
n_components=3
column_index=9
```
the tool will create a vtu output with two scalar fields: one named scar, populated with the 7th column of the alg mesh file, and another scalar field, with 3 components, called fibers (that starts at the 9th column).

I hope this will help.

My best,
Rafael Sachetto

**Reply by LLRiebel** [reply]
Amazing! Thanks Rafael.
